addon/adapters/guid.js

```diff
@@ -0,0 +1,17 @@
+import JSONAPIAdapter from 'ember-data/adapters/json-api';
+
+import config from 'ember-get-config';
+import DataAdapterMixin from 'ember-simple-auth/mixins/data-adapter-mixin';
+
+export default JSONAPIAdapter.extend(DataAdapterMixin, {
+    authorizer: 'authorizer:osf-token',
```

Comment:
*nods* If we want to simplify things, we could always strip down the base class and move the magic into `OsfAdapterDarkArtsMixin`. Composition might be preferable to duplication if it comes to that.